### PR TITLE
Adjust docs url links and add detail license file 

### PR DESCRIPTION
**Pull request by casionone**
### What is the purpose of the change
 Due to the transfer of the git repository to apache and the migration of Linkis-Doc documents to the linkis official website, corresponding links were modified

### Brief change log

-  modify contributing doc
-  add detail license
-  adjust docs url links
-  remote unused test

### Does this pull request potentially affect one of the following parts:
- Dependencies (does it add or upgrade a dependency): (no)
- Anything that affects deployment: (no)
- The MGS(Microservice Governance Services), i.e., Spring Cloud Gateway, OpenFeign, Eureka.: (no)

### Documentation
- Does this pull request introduce a new feature? ( no)

### CODE REVIEW

File: CONTRIBUTING_CN.md [hunk 1]
```diff
@@ -78,7 +78,7 @@ git clone https://github.com/yourname/Linkis.git --branch dev-*
 
 3. 代码目录结构
    
-   详见 [Linkis代码目录结构](https://github.com/WeBankFinTech/Linkis/wiki/Linkis%E6%BA%90%E7%A0%81%E5%B1%82%E7%BA%A7%E7%BB%93%E6%9E%84%E8%AF%A6%E8%A7%A3)
+   详见 [Linkis代码目录结构](https://linkis.apache.org/zh-CN/docs/latest/deployment/sourcecode_hierarchical_structure)
```
Comment (peacewong) [inline CONTRIBUTING_CN.md:81]: This url responds with 404

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ The user configuration is in the project root directory /config/, the project st
 
 3. Code directory structure
    
-   For details, see [Linkis Code Directory Structure](https://github.com/WeBankFinTech/Linkis/wiki/Linkis%E6%BA%90%E7%A0%81%E5%B1%82%E7%BA%A7%E7 %BB%93%E6%9E%84%E8%AF%A6%E8%A7%A3)
+   For details, see [Linkis Code Directory Structure](https://linkis.apache.org/docs/latest/deployment/sourcecode_hierarchical_structure)
```
Comment (peacewong) [inline CONTRIBUTING.md:82]: This url responds with 404

File: README_CN.md [hunk 3]
```diff
@@ -46,26 +46,26 @@ Linkis 自2019年开源发布以来，已累计积累了700多家试验企业和
 
 # 下载
 
-请前往[Linkis releases 页面](https://github.com/WeBankFinTech/Linkis/wiki/Linkis-Releases) 下载Linkis 的已编译版本或源码包。
+请前往[Linkis releases 页面](https://github.com/apache/incubator-linkis/wiki/Linkis-Releases) 下载Linkis 的已编译版本或源码包。
```
Comment (peacewong) [inline README_CN.md:49]: It is recommended to use [Linkis Releases Page](https://github.com/apache/incubator-linkis/releases)

Comment (casionone) [reply README_CN.md]: ok I will fix it

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**